src/db.rs

```diff
@@ -323,6 +329,10 @@ impl Shared {
             }
 
             // The key expired, remove it
+            debug! {
+                key = &key.as_str(),
+                "purged_expired_key",
+            }
```

Comment:
would probably just write
```suggestion
            debug!(key = &key.as_str(), "purged_expired_key");
```